R/mlflow/R/tracking-rest.R

```diff
@@ -0,0 +1,246 @@
+#' List Experiments
+#'
+#' Retrieves MLflow experiments as a data frame.
+#'
+#' @examples
+#' \dontrun{
+#' library(mlflow)
+#' mlflow_install()
+#'
+#' # list local experiments
+#' mlflow_list_experiments()
+#'
+#' # list experiments in remote MLflow server
+#' mlflow_set_tracking_uri("http://tracking-server:5000")
+#' mlflow_list_experiments()
+#' }
+#'
+#' @export
+mlflow_list_experiments <- function() {
+  response <- mlflow_rest("experiments", "list")
+  exps <- response$experiments
+
+  exps$artifact_location <- mlflow_relative_paths(exps$artifact_location)
```

Comment:
R users like working with data frames that are compact since is really common for them to print them in `Rmd` docs, etc. Using relative paths helps them view this list. If they really wanted the full path, they would run something like:

```r
mlflow_list_experiments() %>%
   mutate(artifact_path = normalizePath(artifact_location))
```